docs/api/covidcast_meta.md

```diff
@@ -58,7 +58,7 @@ None.
 | --- | --- | --- |
 | `result` | result code: 1 = success, 2 = too many results, -2 = no results | integer |
 | `epidata` | list of results, one per name/geo_type pair | array of objects |
-| `epidata[].name` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
+| `epidata[].source` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
 | `epidata[].geo_type` | geographic resolution (e.g. `county`, `hrr`, `msa`, `dma`, `state`) | string |
 | `epidata[].min_date` | minimum date (yyyy-MM-dd) | string |
 | `epidata[].max_date` | maximum date (yyyy-MM-dd) | string |
```

Comment:
looks like i neglected to change these to "min/max_time" and to add "min/max_value". i'll do that now.

Reply:
fixed